README.md

```diff
@@ -90,3 +90,28 @@ continuous integration by inspecting `.circleci/config.yml`. Regression tests ar
 performed for a set of reference configurations included in `tests/pytest/config`.
 Please read the README in `tests/pytest` for more information about these regression
 tests and how to update the reference checksums.
+
+# Interactive development of serialized model in Docker
+
+To enter the docker container with the FV3 sources tree mounted to `/FV3/original`,
+run:
+
+    make enter_serialize
+
+When in the docker container, you can reprocess the sources and recompile the model with:
+
+    cd /FV3 && make -C original serialize_preprocess && make libs fv3.exe
+
+If you need to be able to see the preprocessed `atmos_cubed_sphere` sources on your
+host filesystem, you can do that with:
+
+    MOUNTS_SERIALIZE="-v $(pwd)/FV3:/FV3/original -v <local dir>:/FV3/atmos_cubed_sphere" make enter_serialize
+
+Where `<local_dir>` is replaced with the directory on your host filesystem where you
+want to see the preprocessed sources.
+
+This workflow does not support viewing preprocessed sources in the root `/FV3` directory
+on the host filesystem, because `/FV3` has a bind-mounted subdirectory `original` and
+it is not possible to contain a bind-mounted subdirectory in a separately bind-mounted
+directory. You could see these sources by setting `SERIALBOX_OUTDIR` to a different
```

Comment:
Oh good -- I think using SERIALBOX_OUTDIR (for dev) will likely end up being something I do because I still find it more natural to think about the preprocessed files going elsewhere and find this new model a bit confusing. I get the reasons to want to do it, and am not arguing we switch back to what is easier for me, but I like that there is some flexibility.